client/src/components/Layout/AvailableRooms/AvailableRooms.css

```diff
@@ -0,0 +1,4 @@
+* {
+    margin: 0;
+    padding: 0;
```

Comment:
0 is set by default for both margin and padding and neither one gets inherited from parent elements (thus even if parent elements has padding set to 10px, the child will still be set to 0). You can remove this.
Btw. good reference to check (you can see the Initial value and the Inherited properties):
https://developer.mozilla.org/en-US/docs/Web/CSS/margin#Specifications